packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -863,7 +863,10 @@ async function handler(
   const seed = `${organizerUser.username}:${dayjs(reqBody.start).utc().format()}:${new Date().getTime()}`;
   const uid = translator.fromUUID(uuidv5(seed, uuidv5.URL));
 
-  const bookingLocation = getLocationValueForDB(locationBodyString, eventType.locations);
+  const { bookingLocation, bookingLocationCredentialId } = getLocationValueForDB(
```

Comment:
If there is a `credentialId` then it will parse that from the event type location JSON